Program/commandline.h

```diff
@@ -42,7 +42,7 @@ class CommandLine
 	// Reads the line of command and extracts possible options
 	CommandLine(int argc, char* argv[])
 	{
-		if (argc % 2 != 1 || argc > 27 || argc < 3)
+		if (argc % 2 != 1 || argc > 31 || argc < 3)
```

Comment:
after adding "nbIterPenaltyManagement" should it be 33?

Reply:
Yes, thanks for catching that!